src/home_robot_hw/home_robot_hw/nodes/state_estimator.py

```diff
@@ -33,17 +33,28 @@ def cutoff_angle(duration, cutoff_freq):
 
 
 class NavStateEstimator:
-    def __init__(self):
-        # Initialize
+    def __init__(self, trust_slam: bool = False, use_history: bool = True):
+        """Create nav state estimator.
+
+        trust_slam: just use the slam pose instead of odometry.
+        """
```

Comment:
```suggestion
    def __init__(self, trust_slam: bool = False, use_history: bool = True):
        """Create nav state estimator.

        trust_slam: Just use the slam pose instead of odometry.
        use_history: Use previous filtered signals to compute current signal.
        """
```